docs/content/how-to/kubevirt/configuring-network.md

```diff
@@ -58,3 +58,57 @@ hcp create cluster kubevirt \
 --additional-network name:my-namespace/network1 \
 ```
 
+## Avoiding Network CIDR Collisions between Hosting and Guest Clusters.
+
+When creating a guest cluster, it's crucial to ensure that the hosting and guest clusters do not share the same subnet (cidr) for the cluster network.
```

Comment:
When creating a hosted cluster using non external infrastructure mode (sharing the same management cluster).